src/ex1/script.js

```diff
@@ -89,13 +91,13 @@ function filterTodo(event) {
 }
 
 function clearAllTasks() {
-    const allTodos = document.querySelectorAll('.todo')
-    allTodos.forEach(task => {
+    const allTodos = document.querySelectorAll(".todo");
```

Comment:
lines 72,79: export `completed`,`uncompleted` to constants

Reply:
add a const which store true/false if task class contains 'completed'